### DISCUSSION TITLE
FAIMS BoxCar DIA

### DISCUSSION THREAD

**Discussion by cheng01** [vdemichev/DiaNN#116] [Q&A]
Hi developers, wondering does it support FAIMS BoxCar DIA data? Or BoxCar DIA without FAIMS? Thank you very much, Cheng

**Comment by vdemichev** [comment]
Hi Cheng, please see https://github.com/vdemichev/DiaNN/issues/67 for the info on FAIMS. 
BoxCar DIA is supported (it's reported that DIA-NN analyses BoxCar data just fine), but DIA-NN is not optimised for it.
